tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs

```diff
@@ -158,3 +158,23 @@ pub const PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64:
     grouping_factor: LweBskGroupingFactor(3),
     deterministic_execution: false,
 };
+pub const PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64:
```

Comment:
this would likely be better organized in the same way as the classic parameters, such that auto generation by @soonum is easier to plug in, wdyt ?